src/squlearn/qnn/qnnr.py

```diff
@@ -156,15 +156,17 @@ def predict(self, X: np.ndarray) -> np.ndarray:
 
         return self._qnn.evaluate(X, self._param, self._param_op, "f")["f"]
 
-    def partial_fit(self, X: np.ndarray, y: np.ndarray, weights: np.ndarray = None) -> None:
+    def partial_fit(self, X, y, weights: np.ndarray = None) -> None:
```

Comment:
Add doc strings from the fit function array-like or sparse matrix of shape (n_samples, n_features) etc